html-aam/index.html

```diff
@@ -14589,6 +14642,58 @@ <h4 id="att-step">`step`</h4>
             </tr>
           </tbody>
         </table>
+        <h4 id="att-switch">`switch`</h4>
+        <table aria-labelledby="att-switch">
+          <tbody>
+            <tr>
+              <th>HTML Specification</th>
+              <td>`switch`</td>
+            </tr>
+            <tr>
+              <th>Element(s)</th>
+              <td><a data-cite="html/input.html#attr-input-checked">`input`</a> `type=checkbox`</td>
```

Comment:
isn't this missing the switch attr?

Reply:
yeah, the link doesn't exist in the published spec yet.  but looks like it would be `input.html#attr-input-switch` when it does.

Reply:
aw nope, that was just a miss.  already had that link in there for the previous instance.  thanks!